cavern/src/intTest/java/org/opencadc/cavern/VosiCapabilitiesTest.java

```diff
@@ -62,28 +62,42 @@
 *  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
 *                                       <http://www.gnu.org/licenses/>.
 *
-*  $Revision: 4 $
+*  $Revision: 5 $
 *
 ************************************************************************
-*/
+ */
 
 package org.opencadc.cavern;
 
+import ca.nrc.cadc.reg.Capabilities;
+import ca.nrc.cadc.util.Log4jInit;
 import ca.nrc.cadc.vosi.CapabilitiesTest;
-
 import java.net.URI;
-
+import org.apache.log4j.Level;
 import org.apache.log4j.Logger;
 
 /**
- * All tests inherited from CapabilitiesTest
+ *
+ * @author pdowler
  */
-public class VosiCapabilitiesTest extends CapabilitiesTest
-{
+public class VosiCapabilitiesTest extends CapabilitiesTest {
+
     private static final Logger log = Logger.getLogger(VosiCapabilitiesTest.class);
 
-    public VosiCapabilitiesTest()
-    {
-        super(URI.create("ivo://cadc.nrc.ca/arc"));
+    static {
+        Log4jInit.setLevel("org.opencadc.cavern", Level.INFO);
+        Log4jInit.setLevel("ca.nrc.cadc.vosi", Level.INFO);
+        Log4jInit.setLevel("ca.nrc.cadc.reg", Level.DEBUG);
     }
+
+    
+    public VosiCapabilitiesTest() {
+        super(Constants.RESOURCE_ID);
+    }
+
+    @Override
+    protected void validateContent(Capabilities caps) throws Exception {
```

Comment:
It's a reminder that the hook exists so that one can add tests that specific endpoints exist (we do that in other services but was never fleshed out here). It's on the TODO list to add some checks so the test will detect if an endpoint is missing or not correctly describes in the capabilities